docs/cromwell_features/CallCaching.md

```diff
@@ -81,6 +87,38 @@ Cromwell would search cache hits in all of the `gs://alice_bucket`, `gs://bob_bu
 
 If no `call_cache_hit_path_prefixes` are specified then all matching cache hits will be considered.
 
+***Call cache failure logging***
+
+By default when call caching fails to copy previous results the failure reason will be logged. Cromwell will continue to
+search for other call caching candidates, and when no candidates are left Cromwell will run the job on the backend.
+
+The logging of these informational messages can be adjusted via the configuration value:
+
+```hocon
+call-caching {
+  log-cache-hit-failures: true
```

Comment:
Would it be reasonable to simply log these at log level INFO instead of introducing a custom boolean?

"informational messages" seem like a great fit for that log level.

Reply:
The option has been removed.